a11y-meta-display-guide/2.0/draft/guidelines/index.html

```diff
@@ -894,7 +911,40 @@ <h5>Display techniques for prerecorded audio support
 			<section id="wr-examples">
 				<h4>Examples</h4>
 
-				<aside class="example" title="Publication with accessible ways of reading">
+				<aside class="example" title="Publication with text and imges">
+					<p>The following example shows the descriptive and compact statements that would display
+						for a typical text-based publication with images that have text alternatives.</p>
+					
+					<dl>
+						<dt>Compact display</dt>
+						<dd>
+							<p class="ex-hd" data-localization-id="ways-of-reading-title">Ways of reading</p>
+							<p data-localization-id="ways-of-reading-visual-adjustments-modifiable"
+								data-localization-mode="compact">Appearance can be modified</p>
+							<p data-localization-id="ways-of-reading-nonvisual-reading-readable"
+								data-localization-mode="compact">Readable in
+								read aloud or dynamic braille</p>
+							<p data-localization-id="ways-of-reading-nonvisual-reading-alt-text"
+								data-localization-mode="compact">Has alternative text</p>
+						</dd>
+						
+						<dt>Descriptive display</dt>
+						<dd>
+							<p class="ex-hd" data-localization-id="ways-of-reading-title">Ways of reading</p>
+							<p data-localization-id="ways-of-reading-visual-adjustments-modifiable"
+								data-localization-mode="descriptive">Appearance of the text and page layout can be
+								modified according to the capabilities of
+								the reading system (font family and font size,
+								spaces between paragraphs, sentences, words,
+								and letters, as well as color of background
+								and text)</p>
+							<p data-localization-id="ways-of-reading-nonvisual-reading-readable"
+								data-localization-mode="descriptive">All content can be read as read aloud speech or dynamic braille</p>
+						</dd>
```

Comment:
are we missing the "descriptive" version for alt text?
<p data-localization-id="ways-of-reading-nonvisual-reading-alt-text" data-localization-mode="descriptive">Has alternative text descriptions for images</p>

Reply:
Oops, yes, good catch. I cut too much from that one.